### ISSUE TITLE
Logo Survey Roundup

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/opensourcedesign.github.io#28]
The results from the survey are in!

**TL;DR** The new OSD Logo: 
![Interim OSD Logo](http://i.imgur.com/AcpGJXc.png)

[osd-responses.csv](https://gist.github.com/simonv3/006ac966b38526364b82) (edited to remove identifying details) 
## First Round:

| logo option | votes |
| --- | --- |
| 7 | 8 |
| 5 | 5 |
| 23 | 4 |
| 14 | 2 |
| 4 | 2 |
| 27 | 2 |
| None | 2 |
| 3 | 1 |
| 15 | 1 |
| 16 | 1 |
| 8 | 1 |
| 6b | 1 |
| 22 | 1 |
| _total_ | 31 |

And distributing the votes of non-winning options: 

| logo option | votes |
| --- | --- |
| 7 | 9 |
| 5 | 7 |

Which makes 7 by @Xaviju a pretty good winner. 

![Interim OSD Logo](http://i.imgur.com/AcpGJXc.png)

I want to address some of the concerns raised in the survey anonymously: 

> Holy crap that's a lot of choices.

We've got a pretty awesome community :)

> Guys, when you let people choose a logo out of a selection you should at least bring them on the same size or at best in one bigger, one smaller one. Otherwise the comparison might easily be misleading.

This is a valid complaint, but judging by how strapped for time the people working on this already are, I think it's an unfair one. I don't think the logos were that different is size on the logos.html page, people could easily click through to see the original size in #18, and I also trust people to make that judgement for themselves. Though resizing help will always be appreciated in the future :)

> Interim logo

This depends on whether a group of people step forward and want to work on that. Be the change you want to see. I've reached out to the person in the survey who offered their help, and I'm sure we'll post more updates on that soon.

> Current logo preference.

Unfortunately we can't do that. The current logo is a trademark of the Open Source organization. It's not freely available for use.

CC @jancborchardt @GarthDB @una @Xaviju @gillisig @plastelina @MariekevC

**Comment by gillisig** [comment]
Well I am happy with the result! Shall we add a color to it? 

P.s. Lets get some stickers! :smile:

**Comment by mrphs** [comment]
+1 stickers. I love this logo and would proudly wear it on my laptop(s) :)

**Comment by GarthDB** [comment]
Congrats on getting a logo together.

![image](https://cloud.githubusercontent.com/assets/125516/10750703/bdd4f7a6-7c3e-11e5-8438-263ae4f49398.png)

**Comment by razetime** [comment]
Yay! when's the logo gonna change?

**Comment by plastelina** [comment]
Nice! Well done. 
I'll chat in IRC in the afternoon. I've got sticker person ready and waiting to get this printed as well as some pencils, etc.

**Comment by elioqoshi** [comment]
Well done! I suggest to keep the color flexible depending on the medium.

**Comment by elhombretecla** [comment]
Great choice!!!

These stickers will be awesome :)

**Comment by belenbarrospena** [comment]
we made it! We defeated the logo choosing curse ;)

I really like it, and can't wait for the stickers

**Comment by Xaviju** [comment]
Hi!
We have been talking about this in the IRC channel and we decided to make small group (~4 people) to make a meeting to work on this logo to make a **branding** proposal next week (best practices guidelines, stickers...etc.) . So far 3 people joined this working group:

- @plastelina 
- @elhombretecla 
- @xaviju

To make it agile, we would not like to have more than 4 people but if someone really wants to join us and we are already 4 we would make room for five :)
The date and hour will be decided depending on the timezones of the participants.

**How want to join this amazing team?** (Teaming up with @plastelina and @elhombretecla? WOW!)

**Comment by jancborchardt** [comment]
Sounds like a group of 3 is perfect for that, and also enough. Any more will probably lead to conflicts of timing or opinion. ;)

**Comment by plastelina** [comment]
Here is the link to trello board https://trello.com/opensourcedesignlogoguidelinesbranding
- we can start straight away (I've already started some bits and bobs)

Please add yourself and needed boards

@Xaviju - would you mind uploading svg /ai format to the board? we can play with colours and font there then follow up with results in github instead of email chains...

Does that sounds ok?

@Xaviju @elhombretecla

**Comment by Xaviju** [comment]
Taiga project is public now: https://tree.taiga.io/project/xavierjulian-open-source-design-branding/

Ill create a task named logo version with the SVG to play with it
@plastelina @elhombretecla

**Comment by ei8fdb** [comment]
Nice logo!

Could we get some stickers in time for Mozfest on 6th of November? We could hand some out to people there.

I can get some printed in London? I can bring any leftovers to FOSDEM.

Ideas?

------------
Bernard Tyers
E: ei8fdb@ei8fdb.org | T: @bernardtyers

Talk to me privately. My public key is here: https://keybase.io/ei8fdb

**Comment by bnvk** [comment]
Great job everyone! Quite happy to have a proper logo reached
through a wonderful civil consensus :D

**Comment by jancborchardt** [comment]
I'd say for the sticker a circle would look best, with white background and white border around like the typical ones from http://stickermule.com, what do you think?

**Comment by elhombretecla** [comment]
Agree with Jan. Stickermule works really fine and 5x5 cm stickers made with new Logo + white border sounds good :)

**Comment by simonv3** [comment]
Woop - super happy about the reaction to this! Thanks for being so excited about all of this folks. :+1: for Stickermule for insta-logo stickers.

Also @GarthDB, I approve.

**Comment by ei8fdb** [comment]
In terms of design (circle with white background, white border, 5cm x 5cm) it sounds good to me.

Stickermule is approx. £40 for 100. Zazzle.co.uk is £18 for 100 (3.8cm diameter).

I’ve had Zazzle stickers before and they’re fine. Is Stickermule “worth” the difference in cost?

Just wonderin…

Bernard
------------
Bernard Tyers
E: ei8fdb@ei8fdb.org | T: @bernardtyers

Talk to me privately. My public key is here: https://keybase.io/ei8fdb

**Comment by jancborchardt** [comment]
I’d also say that the 3.8 cm diameter from Zazzle is slightly better than the full 5cm. We don’t want to »make the logo bigger«, eh? ;D

**Comment by ei8fdb** [comment]
Heh.

(Apologies for the ugly URL..it’s auto-generated)

Here is the shopping cart on Zazzle so you can see the details:

http://www.zazzle.co.uk/co/cartrc?kt=AQBlAAAAAQAAABTLc_6Q6sbp9O2lixV_affIFpGIhP_YS1eQ3FKGJo_0XX_bRzdl_4B7yxuK6QDUCLdstdmNDeKAK0Kivu_cqjVmukPVn8ELzEZwRf7eWzuIGRcRzrd_sjIYlfP00mCYeA2HENc5Wgw7Ui0aIWAvTPpJsoD6rA&CMPN=emc_en-gb_share_cart_button1&rf=238625503972086358




------------
Bernard Tyers
E: ei8fdb@ei8fdb.org | T: @bernardtyers

Talk to me privately. My public key is here: https://keybase.io/ei8fdb

**Comment by ei8fdb** [comment]
Hmm, but looking at the quality comparison reviews between them it seems Stickermule provides much better quality.

Anyone have experience of both?

------------
Bernard Tyers
E: ei8fdb@ei8fdb.org | T: @bernardtyers

Talk to me privately. My public key is here: https://keybase.io/ei8fdb

**Comment by plastelina** [comment]
Morning!

I do have experience with both companies, yes Stickermule is very good but it's being shipped from US and A and I don't think you will get it in time, the Zazzle is pricey and not good.

I've used Stickermule and printed.com, both are great.

https://www.printed.com/order-wizard/product-options/round-stickers
I've chose 25mm, one side, silver gloss, 170 gsm paper, 100 stickers and it's 15 quid. I would rather go for this. You do not need to wait for me to prepare the design, just upload SVG and you are done, unless you want more colours, then I will be ready this evening/tomorrow the latest.

I was planing to order from them (printed.com) for FOSDEM anyway. 

Up to you.

**Comment by Xaviju** [comment]
Here you have the default SVG if you want to print your own copy but we are working on it and there will be SVG versions for sticker soon. 
SVG: https://media.taiga.io/attachments/f/9/2/5/04e97ac5b7d30e3af5cf5f9c02200937e4b7fed4a41a66d39b0ff4a86cb0/opensourcedesign.svg

**Comment by jancborchardt** [comment]
@plastelina sounds good!

**Comment by plastelina** [comment]
Bad news, spilled coffee on my laptop while working today. Waiting to get it fixed. Will be ready with all the guidelines, etc after weekend....so annoyed! 
In the meantime...if you need stickers you can just use the svg file and upload it to the order.

**Comment by ei8fdb** [comment]
Printed look a good price.

Is there any objection to me trying to use the current SVG and print a few hundred for Mozfest?

If anyone is going I’ll give you a bunch too :)

**Comment by simonv3** [comment]
I don't have any objections, I don't see why anyone would. Please go for it - let's advertise!

**Comment by jancborchardt** [comment]
Yeah, just go for it! We can even try out different printers on our own and then compare which is the best one to use that for future stickers. Won’t be the last batch we print. ;)

**Comment by ei8fdb** [comment]
OK, I’ll send off for a batch today.

**Comment by simonv3** [comment]
Hey @plastelina @Xaviju @elhombretecla will you let us know when it's okay to change our website logo and such?

**Comment by plastelina** [comment]
Hi @simonv3, I guess you can update it now. There is no reason why not. My laptop is still in apple getting fixed so I can deliver all the document sometime this week...not sure when. So frustrating.

**Comment by simonv3** [comment]
@Xaviju, could you post an svg version of the logo?

**Comment by Xaviju** [comment]
@simonv3 Sure, here -> https://www.dropbox.com/s/19pv1vsg2boa92s/opensourcedesign.svg?dl=0

**Comment by simonv3** [comment]
Great, thanks, and thanks for the quick reply. I'll commit it to the repo with the work I'm doing right now :)
